appium/tests/specs/composeEmail/SendEncryptedEmailAfterWrongPassPhrase.spec.ts

```diff
@@ -23,10 +23,15 @@ describe('COMPOSE EMAIL: ', () => {
     const wrongPassPhraseError = CommonData.errors.wrongPassPhrase;
     const wrongPassPhrase = "wrong";
     const senderEmail = CommonData.account.email;
+    const bundleId = CommonData.bundleId.id;
 
     await SplashScreen.login();
     await SetupKeyScreen.setPassPhrase();
 
+    //Restart app to reset pass phrase memory cache
```

Comment:
@tomholub
This does not work and I have no idea how to fix it. It should restart app because when we did `SetupKeyScreen.setPassPhrase()` we need to reset in memory pass phrase cache.

Reply:
I'll take a look

Reply:
I'll take a look

Reply:
Did you run this locally while watching the simulator? Here's what happens:
 - app is started, authenticated, then pass phrase entered for setup
 - app is restarted
 - app is showing splash screen again - not set up

One could deduce that very likely, we are restarting the app too soon, before it had a chance to write setup result to storage. That's why we are seeing a splash screen again.

Don't you think? I'll try to add a wait for loaded inbox, see if that fixes it.

Next time you run into similar issue, please share a video of the failing test - you will get quicker help because sometime the problem may be obvious without having to rebuild and run it.

Reply:
That makes me think - it would be good if the CI produced a video of failed tests as well. I'll file an issue.

Reply:
Maybe I'm wrong but on my side it was like reinstall not just launching the same instance

Reply:
Hm. Try to run it after the changes I just pushed

Reply:
@tomholub
You was right about delay, it works. I will pay more attention on such things in future.

Reply:
Didn't fix it with a delay - fixed with awaiting the next expected step before restarting (waiting for rendered inbox screen)